app/constants/radioButtons.ts

```diff
@@ -0,0 +1 @@
+export const YES = 'Yes';
```

Comment:
implement with enum for both options - `RadioButtonOption.Positive` & `RadioButtonOption.Negative`